ROLES.md

```diff
@@ -224,15 +193,7 @@ to support users and keep test quality high, as well as release leads
 [rotation](https://github.com/knative/pkg/blob/main/RELEASE-LEADS.md) to
 shepherd Knative releases.
 
-# Working Group Leadership Roles
-
-We differentiate here between two key roles, WG Execution Lead and WG Technical
-Lead. In small working groups, often these roles will be performed by one
-person; in larger working groups it might be best to have different individuals
-performing these roles. There is no 'formula', it is up to the working group
-leadership to determine what is best for their working group.
-
-## Working Group Technical Lead
+## Working Group Lead
 
 Working group technical leads, or just ‘tech leads’, are approvers of an entire
```

Comment:
```suggestion
Working group leads are approvers of an entire
```

It looks like we're dropping the distinction between "execution leads" (doing an executive role of organizing work) and "technical leads" (primarily focused on code).  That seems reasonable, given the current size/state of the project.

Reply:
The distinction seems aspirational and not practical, imo.